src/database/models/event.model.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @author Herobiam Heita
+ * @email hherobiam@gmail.com
+ * @create date 2022-07-02 19:03:46
+ * @modify date 2022-07-02 19:03:46
+ * @desc [description]
+ */
+import { Schema, model } from 'mongoose';
+
+//import ai from 'mongoose-auto-increment';
+
+export const EventRawType = {
+  _id: { type: String, required: true, alias: 'id' },
+  name: { type: String, required: true },
+  description: { type: String, required: true },
+  host: { type: Array, required: false },
+  category: { type: Array, required: false },
+  eventDate: { type: String, required: false },
+  endTime: { type: String, required: false },
+  location: { type: Array, required: false },
+  attendance: { type: Number, required: false },
+  tickets: { type: Array, required: false },
+  images: { type: String, required: false },
+};
+
+export interface EventSchemaType {
+  _id: string;
+  name: string;
+  description: string;
+  host: any;
```

Comment:
fields with any will have to change to specific models when all of them are created, e.g host will be of type Host

Reply:
Yes this will any will change host type in the future, as we discussed we need to meet to make sure definitions are okay